public/app/screens/main/sections/devices/device-state/device-state.jsx

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+var Link = require('react-router').Link;
+var styleMixin = require('mixins/style-mixin');
+var DeviceState = React.createClass({
+  mixins: [styleMixin(require('./style.scss'))],
+
+  render: function render () {
+    return (
+        <div className="cc-device-state">
+          <div className={'cc-device-state__image ' + this.getStateImageClass()}/>
```

Comment:
There are  2 functions introduced  for it. We need to use the same tools for solving similar problems.